client/client.go

```diff
@@ -41,6 +41,7 @@ const (
 	errMissingCerts = "No trusted certificates. The client will continue running, but will " +
 		"not be able to communicate with the server. Either specify ServerCertificate in " +
 		"mender.conf, or make sure that CA certificates are installed on the system"
+	pkcs11URIPrefix = "pkcs11:"
```

Comment:
I thought it is nice. I can remove, but if Lluis agrees (see [above](https://github.com/mendersoftware/mender/pull/598#discussion_r479075246)) it will be used in the `Validate` as well.